assets/userCtrl.js

```diff
@@ -285,13 +289,20 @@ module.exports = function ($scope, $location, io) {
       var game = $scope.user.flairGames[j];
       if (game.ign) {
         hasIGN = true;
+        if (game.ign.length > 12) {
```

Comment:
We should probably put this in the shared function for isCorrectFlairText, so that the backend does it as well, and we should probably use that function here instead of having it twice. I had thought I had done that in the past. But apparently not...